login-fire-button.html

```diff
@@ -195,16 +195,17 @@
     },
 
     /**
-     * The provider logo must be hidden if the element can sign out and a user
-     * is currently signed in.
+     * The provider logo is shown when the user can sign in and is not signed in
+     * Or when they can not sign out. It is hidden otherwise.
```

Comment:
I guess it's not necessarily right or wrong but I try to not assume the user's gender. https://en.oxforddictionaries.com/usage/he-or-she-versus-they

We can discuss further when we do #69 where we will try to go through all the documentation and redo it.